instrumentation/resque/lib/opentelemetry/instrumentation/resque/instrumentation.rb

```diff
@@ -18,8 +18,38 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
           defined?(::Resque)
         end
 
-        option :span_naming,       default: :queue, validate: %I[job_class queue]
-        option :propagation_style, default: :link,  validate: %i[link child none]
+        ## Supported configuration keys for the install config hash:
+        #
+        # force_flush: controls if spans are forcibly flushed upon job completion
+        #   - :ask_the_job (default) - if `Resque::Worker#fork_per_job?` is set,
+        #     all completed spans will be synchronously flushed at the end of a
+        #     job's execution
+        #   - :always - all completed spans will be synchronously flushed at the
+        #     end of a job's execution
+        #   - :never - the job will not intervene with the processing of spans
+        #
+        # when `true`, all completed spans will be synchronously flushed
+        #   at the end of a job's execution (default: `false`). You may wish to
+        #   enable this option if `Resque::Worker#fork_per_job?` is enabled.
```

Comment:
What do you think of including the reason the reader may wish this?

I _think_ the recommendation is because the spans are queued in a processor+exporter instance that lives in a background thread of the forked worker process and so without force flushing, the spans there are at likely to remain unsent due to batch timing and then discarded when Sidekiq ends the forked process.

… And this pairs with my question about "when `true` …" at the beginning of this paragraph. Would I set `force_flush` to `:ask_the_job` or to `true` for the synchronous, blocking send behavior?